Sources/RingPublishingTracking/Private/Networking/Endpoints/Responses/ArthemisResponse.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  ArtemisResponse.swift
+//  RingPublishingTracking
+//
+//  Created by Adam Mordavsky on 09.11.23.
+//
+
+import Foundation
+
+struct ArtemisResponse: Decodable {
+
+    struct CFG: Decodable {
+        /// Expiration time
+        let ttl: Int
+    }
+
+    struct User: Decodable {
+
+        let id: ID
+    }
+
+    struct ID: Decodable {
+        let real: String
+
+        let model: String
+
+        let models: Models
```

Comment:
I just wonder if this should be handled as blackbox or not - I will aks Michał for confirmation